components/batch-system/src/batch.rs

```diff
@@ -391,7 +392,12 @@ where
             };
             let t = thread::Builder::new()
                 .name(thd_name!(format!("{}-{}", name_prefix, i)))
-                .spawn(move || poller.poll())
+                .spawn(move || {
+                    if run_with_high_pri {
+                        let _ = sys::thread::set_priority(sys::HIGH_PRI);
+                    }
```

Comment:
Why not always set to high priority?

Reply:
I think we shall move this to method `poll`. In other words, we always set to hight priority in method `fn poll` of `ApplyPoller` and `RaftPoller`.